src/placement-center/src/raftv1/snapshot.rs

```diff
@@ -52,7 +52,7 @@ impl RaftSnapshot {
         Ok(())
     }
 
-    #[warn(dead_code)]
+    #[allow(dead_code)]
```

Comment:
Similarly, is this supposed to be #[allow(dead_code)] instead of #[warn(dead_code)]?